docs/kss/starter-pages/details-1.html

```diff
@@ -0,0 +1,321 @@
+<!DOCTYPE html>
+<head>
+    <title>Techne: Layout Template - Details</title>
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="../public/dist/css/techne.min.css">
+    <link rel="stylesheet" href="css/demo.css">
+</head>
+<body>
+
+    <!-- page header  -->
+    <div class="navbar navbar-default y-navbar-centered y-navbar-lg">
+        <div class="container-fluid">
+            <div class="navbar-left">
+                <div class="y-toolbar">
+                    <div class="y-toolbar-cell">
+                        <button class="btn-subordinate btn navbar-btn">
+                        <span class="hyicon hyicon-back"></span> <span class="hidden-xs">Back</span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <div class="y-navbar-center">
+                <div class="y-navbar-center-content">
+                    <h1>Page Title</h1>
+                </div>
+            </div>
+            <div class="navbar-right">
+            </div>
+        </div>
+    </div>
+
+    <!-- page level help -->
+    <span class="page-level-help-message">Page level help message goes here</span>
+
+    <!-- top panel -->
+    <div class="panel panel-default">
+        <div class="panel-body">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-4">
+                        <div class="y-thumbnail y-thumbnail--circle ng-scope" align="center">
+                            <img src="images/project-icon.png">
+                        </div>
+                    </div>
+                    <div class="col-md-8">
+                        <div class="row">
+                            <div class="col-md-12 form-readonly-text form-readonly-text-lg">
+                                <span class="badge badge-success">GOOD</span>
+                            </div>
+                            <div class="col-md-6">
+                                <div class="form-group">
+                                    <label>Label</label>
+                                    <p class="form-readonly-text form-readonly-text-lg">non editable value</p>
+                                </div>
+                                <div class="form-group">
+                                    <label>Label</label>
+                                    <input type="text" class="form-control input-lg">
+                                    <span class="help-inline help-inline--tooltip">
+                                    <span class="hyicon hyicon-helpoff"></span>
+                                    <span class="help-block-inline ng-binding">Additional help message goes here</span>
+                                    </span>
+                                    <span class="help-inline {$modifiers}">
```

Comment:
the indentation here for the children needs to be fixed. It's not clear that the elements inside of this page are children as they look like siblings